### PR TITLE
Update strings and fwlinks

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

File: Extension/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "cpptools",
   "displayName": "C/C++",
   "description": "C/C++ IntelliSense, debugging, and code browsing.",
-  "version": "0.28.2-master",
+  "version": "0.28.3-master",
```
Comment (Colengms) [inline Extension/package.json]: Should this be 0.29.0, or do we anticipate another patch?

Comment (sean-mcmanus) [reply Extension/package.json]: I anticipate it will be 0.29.0, but I didn't want to increase it too early. However, it should be fine to use 0.29.0, and then we could still decrease it later if there's a reason to.

**Review by Colengms — APPROVED** [review]
